test-network/prometheus-grafana/grafana/provisioning/dashboards/hlf-performances.json

```diff
@@ -0,0 +1,2407 @@
+		  "{id=\"/\",instance=\"cadvisor:8080\",job=\"prometheus\"}": "#BA43A9"
+		},
+		"bars": false,
+		"dashLength": 10,
+		"dashes": false,
+		"datasource": "Prometheus",
+		"editable": true,
+		"error": false,
+		"fill": 1,
+		"fillGradient": 0,
+		"grid": {},
+		"gridPos": {
+		  "h": 6,
+		  "w": 5,
+		  "x": 0,
+		  "y": 0
+		},
+		"hiddenSeries": false,
+		"id": 5,
+		"legend": {
+		  "avg": false,
+		  "current": false,
+		  "max": false,
+		  "min": false,
+		  "show": false,
+		  "total": false,
+		  "values": false
+		},
+		"lines": true,
+		"linewidth": 1,
+		"links": [],
+		"nullPointMode": "null as zero",
+		"options": {
+		  "alertThreshold": true
+		},
+		"percentage": false,
+		"pluginVersion": "8.2.2",
+		"pointradius": 5,
+		"points": false,
+		"renderer": "flot",
+		"seriesOverrides": [],
+		"spaceLength": 10,
+		"stack": true,
+		"steppedLine": false,
+		"targets": [
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_system_seconds_total[1m]))/8",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "a",
+			"refId": "B",
+			"step": 120
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_system_seconds_total{name=~\".+\"}[1m]))/8",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "nur container",
+			"refId": "F",
+			"step": 10
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_system_seconds_total{id=\"/\"}[1m]))/8",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "nur docker host",
+			"metric": "",
+			"refId": "A",
+			"step": 20
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(process_cpu_seconds_total[$interval]))/8 * 100",
+			"hide": true,
+			"interval": "",
+			"intervalFactor": 2,
+			"legendFormat": "host",
+			"metric": "",
+			"refId": "C",
+			"step": 600
+		  },
+		  {
+			"expr": "sum(rate(container_cpu_system_seconds_total{name=~\".+\"}[1m])) + sum(rate(container_cpu_system_seconds_total{id=\"/\"}[1m])) + sum(rate(process_cpu_seconds_total[1m]))",
+			"hide": true,
+			"intervalFactor": 2,
+			"legendFormat": "",
+			"refId": "D",
+			"step": 120
+		  },
+		  {
+			"exemplar": true,
+			"expr": "sum(rate(container_cpu_usage_seconds_total{name=~\".+\"}[$interval]))/8 * 100",
+			"hide": false,
+			"interval": "",
+			"legendFormat": "",
+			"refId": "E"
+		  }
+		],
+		"thresholds": [],
+		"timeFrom": null,
+		"timeRegions": [],
+		"timeShift": null,
+		"title": "CPU Usage",
+		"tooltip": {
+		  "msResolution": true,
+		  "shared": true,
+		  "sort": 0,
+		  "value_type": "cumulative"
+		},
+		"type": "graph",
+		"xaxis": {
+		  "buckets": null,
+		  "mode": "time",
+		  "name": null,
+		  "show": false,
+		  "values": []
+		},
+		"yaxes": [
+		  {
+			"$$hashKey": "object:7341",
+			"format": "percent",
+			"label": "",
+			"logBase": 1,
+			"max": null,
+			"min": null,
+			"show": true
+		  },
+		  {
+			"$$hashKey": "object:7342",
```

Comment:
I would add block height to the dashboard. It is one of the more useful and easy to understand metrics.